qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -153,7 +153,7 @@ def test_full_health(self):
                         'max_avail': int,
                         'quota_objects': int,
                         'quota_bytes': int,
-                        'dirty': int,
+                        'dirty': JAny(none=True),
```

Comment:
what is this change for ? since we always dump an integer `dirty`, there is no point of using `JAny` here, i think.

Reply:
You're right @tchaikov. The thing is that Dashboard custom 'schema' framework doesn't have anything like `JUnion([int, None])` (or `Optional[int]` in Python `typing` syntax).

Reply:
thanks @tchaikov, repushed, let's see if nothing breaks because of schema changes, that would be ideal